planet4-gutenberg-blocks.php

```diff
@@ -116,6 +116,7 @@
 	'planet4-blocks/timeline',
 	'leadin/hubspot-form-block',
 	'gravityforms/form',
+	'planet4-blocks/articles',
```

Comment:
Since the rest of the items here are ordered alphabetically, maybe we can move articles to its alphabetical place.